packages/ds-medicare-gov/src/components/HelpDrawer/HelpDrawer.stories.tsx

```diff
@@ -59,7 +59,7 @@ const drawerContent = (
 );
 
 export const HelpDrawerToggleWithDrawer: Story = {
-  render: function Component(args) {
+  render: function Component() {
```

Comment:
I went down a rabbit hole of cleaning up these stories trying to get the arg tables to show up. Right now this medicare-specific story shows either a table with bogus args or no table at all. Both the main `HelpDrawer` and this Medicare-specific one need work, but I'm spending too many cycles on it right now and need to move on. Would you mind reverting this module's changes for now so the CI tests pass, and you can merge?

Reply:
I'll create a follow-up PR at a future date

Reply:
@dmethvin-gov, the current CI failure is due to these modules changes. If you revert this module to what's on `main`, they should pass. I'll clean these up at some point.

Reply:
That is super strange, since they aren't used. However, it did indeed fix the problem locally?

Reply:
[Here's the follow-up](https://github.com/CMSgov/design-system/pull/2902)